concourse/scripts/pipelines-bosh-cloudfoundry.sh

```diff
@@ -39,6 +40,12 @@ prepare_environment() {
   cf_graphite_version=$("${SCRIPT_DIR}"/val_from_yaml.rb releases.graphite.version "${cf_manifest_dir}/055-graphite.yml")
   cf_grafana_version=$("${SCRIPT_DIR}"/val_from_yaml.rb releases.grafana.version "${cf_manifest_dir}/055-graphite.yml")
 
+  if ${COMMIT_VERIFICATION} ; then
```

Comment:
Will this do the right thin if `VERIFY_COMMITS` is set to `"false"`?  I wonder whether it would be better to invert the sense of this variable - ie name it `SKIP_COMMIT_VERIFICATION`, and then test for empty string here. This would be more consistent with other toggles we have in place.